homeassistant/components/homematicip_cloud/alarm_control_panel.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, hap: HomematicipHAP, security_zones) -> None:
         for security_zone in security_zones:
             if security_zone.label == "INTERNAL":
                 self._internal_alarm_zone = security_zone
-            else:
+            elif security_zone.label == "EXTERNAL":
                 self._external_alarm_zone = security_zone
```

Comment:
Looks like these instance attributes may not always be initialized. All instance attributes should always be initialized in init method at least to `None`.

Reply:
Done